distribution/tools/server-cli/src/test/java/org/elasticsearch/server/cli/ServerCliTests.java

```diff
@@ -48,11 +48,15 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/93335")
 public class ServerCliTests extends CommandTestCase {
 
     private SecureSettingsLoader mockSecureSettingsLoader;
 
+    @BeforeClass
+    public static void skipForFips() {
+        assumeTrue("Pending fix for https://github.com/elastic/elasticsearch/issues/93335", inFipsJvm());
```

Comment:
I think this condition is reversed, we want to assume we are not in a fips jvm right?